### PR TITLE
Contact form and its styling

### PR DESCRIPTION
**Pull request by tahseenzahra**
This branch contains the contact form in the portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,220 +1,285 @@
...
-</ul>
-</section>
+          </ul>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="footer">
+      <div class="Interfont footerTxt">
+        I'm always interested in hearing about new projects, so if you'd like to
```
Comment (zainsadaqat) [inline index.html:236]: #### According to Figma design, this paragraph should be center-aligned. Also, please replace `<div>` tag with `<p>` as you know it's a paragraph and it should be written in `<p>` tag.

File: index.html [hunk 2]
```diff
@@ -1,220 +1,285 @@
...
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/xeqnpapb"
+        method="POST"
+      >
+        <input
+          class="no-outline"
+          type="text"
+          maxlength="30"
+          placeholder="Full Name"
```
Comment (zainsadaqat) [inline index.html:248]: #### This input field is required. Please add the required attribute

File: index.html [hunk 3]
```diff
@@ -1,220 +1,285 @@
...
+        />
+
+        <input
+          class="no-outline"
+          type="email"
+          name="email"
+          placeholder="Email Address"
```
Comment (zainsadaqat) [inline index.html:255]: #### This input field is required. Please add the required attribute

File: index.html [hunk 4]
```diff
@@ -1,220 +1,285 @@
...
+        />
+
+        <textarea
+          name="message"
+          maxlength="500"
+          rows="7"
+          cols="70"
+          class="formTxtArea"
```
Comment (zainsadaqat) [inline index.html:263]: #### This input field is required. Please add the required attribute

File: index.html [hunk 5]
```diff
@@ -1,220 +1,285 @@
...
+      >
+        <input
+          class="no-outline"
+          type="text"
+          maxlength="30"
+          placeholder="Full Name"
+        />
+
+        <input
+          class="no-outline"
```
Comment (zainsadaqat) [inline index.html:252]: Kindly add some bottom padding to make it look clean and near to the required Figma design.

File: index.html [hunk 6]
```diff
@@ -1,220 +1,285 @@
...
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </div>
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/xeqnpapb"
+        method="POST"
+      >
+        <input
+          class="no-outline"
```
Comment (zainsadaqat) [inline index.html:245]: Kindly add some bottom padding to make it look clean and near to the required Figma design.

File: index.html [hunk 7]
```diff
@@ -1,220 +1,285 @@
...
+        />
+
+        <textarea
+          name="message"
+          maxlength="500"
+          rows="7"
+          cols="70"
+          class="formTxtArea"
+        >
+        Enter text here.</textarea
```
Comment (zainsadaqat) [inline index.html]: #### Please add a placeholder for `<textarea>` instead of hard-coded text 👍🏻

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello Team! 😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -246,23 +246,23 @@ <h3 class="mainText CreteRoundfont">About me</h3>
...
           maxlength="500"
           rows="7"
           cols="70"
           class="formTxtArea"
-        >
-        Enter text here.</textarea
+          placeholder="Enter text here."
+        required></textarea
         >
         <button type="button" class="PrjBTN SePrjBTN Interfont">
```
Comment (xtrahuman) [inline index.html]: Your form is currently not making a post request or submitting, try changing the type button to type submit, It probably might fix the issue.

File: index.html [hunk 9]
```diff
@@ -232,10 +232,10 @@ <h3 class="mainText CreteRoundfont">About me</h3>
     </section>
 
     <footer class="footer">
-      <div class="Interfont footerTxt">
+      <p class="Interfont footerTxt">
```
Comment (xtrahuman) [inline index.html:235]: Nice job implementing the changes, though I noticed the text is no longer in the center of the page, you can apply `width: 100% `and `text-align: center` to your `p`  to fix

This is how your text current appears. kindly fix as suggested

image

**Review by xtrahuman — CHANGES_REQUESTED** [review]
Hello @tahseenzahra, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] clean and organized code

## Required Changes ♻️
_Check the comments under the review._ ⬇️ ⬇️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 10]
```diff
@@ -23,6 +23,14 @@ View the file through the 'index.html'.
 - Twitter: [@twitterhandle](https://twitter.com/tahseen1zahra)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/tahseenzahra/)
 
+👤 **Author**
```
Comment (gtekle) [inline README.md]: - [ Optional ] You should use the actual name of the author on behalf of the placeholder text (**Author**)

File: index.html [hunk 11]
```diff
@@ -1,220 +1,285 @@
...
+        <textarea
+          name="message"
+          maxlength="500"
+          rows="7"
+          cols="70"
+          class="formTxtArea"
+          placeholder="Enter text here."
+        required></textarea
+        >
+        <button type="submit" class="PrjBTN SePrjBTN Interfont">
```
Comment (gtekle) [inline index.html]: - [ Optional ] You should align this button horizontally to center as in the figma design.

**Review by gtekle — APPROVED** [review]
Hi **@TEAM**, 
# STATUS: APPROVED :clap: :clap: 🎉 🎉 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
clap-clapping


## Highlights
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Meaningful commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — COMMENTED** [review]
## Hi @tahseenzahra, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the PR is already approved and merged please create a new PR with all the changes. Good luck 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**